src/pages/ListSwaps/SwapRow.tsx

```diff
@@ -130,12 +133,21 @@ function SwapRow(swap: Swap) {
       );
     });
 
+  // TODO: Default values could come from config file or MetaMask
   const acceptFields = [];
   if (!alphaIsTransitive) {
-    acceptFields.push("alpha_ledger_redeem_identity");
+    acceptFields.push({
+      key: "alpha_ledger_redeem_identity",
+      label: "Alpha Redeem Identity",
+      default: "0x493A5A0EaFDEaE28f430F74Ac5031b3eE37B6a6D"
```

Comment:
Ähm, :x: ? :D 

Are these values filled in by default or what is this doing here?

Reply:
Oh, this was just for development convenience.